apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -720,6 +734,22 @@ export class WorkshopForm extends React.Component {
     return value;
   };
 
+  // Selects the given value in the topic dropdown
+  handleSelect = event => {
```

Comment:
Nit: Would you mind adding a more descriptive function name please? Just in case we end up adding other Selects

Reply:
Updated! Thanks!